code/modules/wod13/wallcode.dm

```diff
@@ -597,7 +597,7 @@
 /turf/open/floor/plating/parquetry
 	gender = PLURAL
 	name = "parquetry"
-	icon = 'code/modules/wod13/tiles.dmi'
+	icon = 'modular_zapoc/tiles.dmi'
```

Comment:
This is a lot of repeated code. Make a parent subtype for a base tile that will be used a lot. Give it all the base features so you aren't writing gender, icon, flags_1, and attachement_holes

Reply:
gggggghhhhhhhhh